tests/default/_core/search/query/geo_bounding_box.yaml

```diff
@@ -0,0 +1,60 @@
+$schema: ../../../../../json_schemas/test_story.schema.yaml
+
+description: Test search endpoint with geo_bounding_box query.
+prologues:
+  - path: /point_index
```

Comment:
I missed this one, but would be nice to use the cinema/movie/books here too.

Reply:
Done https://github.com/opensearch-project/opensearch-api-specification/pull/562